index.md

```diff
@@ -0,0 +1,220 @@
+## Styleguide
+
+### Table of Contents
+
+- [1.0 General](#10-general)
+- [2.0 Angular / Coffeescript](#20-angular--coffeescript)
+  - [2.1 Basics](#21-basics)
+  - [2.2 Angular specifics](#22-angular-specifics)
+  - [2.3 Coffeescript specifics](#23-coffeescript-specifics)
+- [3.0 Django / Python](#30-django--python)
+  - [3.1 Basics](#31-basics)
+- [4.0 Styleguide](#40-styleguide)
+
+
+### 1.0 General
+
+- Avoid unnecessary else statements
+
+```python
+# bad
+if a_true_thing:
+    return True
+else:
+    return False
+
+# good
+if a_true_thing:
+    return True
+return False
+```
+
+- Avoid nested if statements
+
+```python
+# bad
+if a_true_thing:
+    if another_true_thing:
+        return 'foo'
+    else:
+        return 'bar'
+
+# good
+if a_true_thing and another_true_thing:
+    return 'foo'
+elif a_true_thing:
+    return 'bar'
+```
+
+### 2.0 Angular / Coffeescript
+
+#### 2.1 Basics
+
+- 2 space indent
+
+```coffeescript
+hello = ->
+  console.log('hi')
+  return
+hello()
+```
+
+- Single quotes for strings, unless using string interpolation, then double quotes is acceptable
+
+```coffeescript
+# bad
+aString = "Hello World"
+anotherString = '#{ aString }' # this won't work at all
+
+# good
+aString = 'Hello World'
+anotherString = "#{ aString }"
```

Comment:
nit (:trollface:): add some more to the interpolated string. The way you have it looks like you should only do this for when the string is entirely an interpolated variable.
```coffeescript
anotherString = "Hello World #{ aString }"
```